ncm-spma/src/main/perl/spma/dnf.pm

```diff
@@ -125,7 +125,15 @@ sub generate_dnf_modules {
         my $fh = $trd->filewriter("$directory/$name.yaml",
                                   header => "# File generated by " . __PACKAGE__ . ". Do not edit",
                                   log => $self);
-        $changes += $fh->close() || 0; # handle undef
+        $module_changed += $fh->close() || 0; #handle undef
+        $changes += $module_changed; 
+        if ($module_changed > 0 and !$CAF::Object::NoAction) {
+            my ($cmd_exit, $cmd_out, $cmd_err) = $self->execute_command([DNF_MODULE_RESET, $name], "module stream change detected. Reset managed module stream ".$name." for update", 1, "/dev/null", 1);
```

Comment:
This line is unreadably long, can you break it up so it wraps a bit more easily please? Also I don't see any reason not to use string interpolation here? 

"module stream change detected. Reset managed module stream $name for update"

Reply:
updated

Reply:
and yes I believe DNF provider is not used by anyone else. We started using on rhel8 builds.